engine/clients/redis/search.py

```diff
@@ -64,7 +64,7 @@ def search_one(cls, vector, meta_conditions, top) -> List[Tuple[int, float]]:
         params_dict = {
             "vec_param": np.array(vector).astype(np.float32).tobytes(),
             "K": top,
-            "EF": cls.search_params["search_params"]["ef"],
+            "EF": cls.search_params["config"]["ef"],
```

Comment:
```suggestion
            **cls.search_params["config"],
```

Also rename `ef` to `EF` in `redis-single-node.json`

Reply:
OK, "EF" it is then.